requirements/pytorch/extra.txt

```diff
@@ -5,6 +5,6 @@
 matplotlib>3.1, <3.6.2
 omegaconf >=2.0.5, <2.4.0
 hydra-core >=1.0.5, <1.4.0
-jsonargparse[signatures] >=4.18.0, <4.23.0
+jsonargparse[signatures] @ https://github.com/omni-us/jsonargparse/zipball/issue-170-class-instantiator
```

Comment:
Note that a feature that does not yet exist in jsonargparse is used, which is why for now the requirement has been changed to a branch.

Reply:
Do you have an ETA for this to be released, bc we shall not release it until then as PyPU+I does not allow dependencies outside PyPI registry

Reply:
There is no plan yet. This feature might change based on the discussion here. When there is more clarity about this I will finalize the jsonargparse feature and change this line to a release.

Reply:
This feature is now part of jsonargparse 4.24.0.